src/Helper/PaymentMath.php

```diff
@@ -31,6 +31,18 @@ class PaymentMath
      */
     private static $useBcMath = true;
 
+    /**
+     * @return int
+     */
+    protected static function getPrecision()
+    {
+        $precision = (int)self::config()->get('precision');
+        if ($precision < 0) {
+            $precision = 0;
+        }
+        return $precision;
```

Comment:
Added this method to address this failure: https://app.travis-ci.com/github/silverstripe/silverstripe-omnipay/jobs/546510897#L1443-L1444. PHP 8 now errors with a negative precision